gensim/models/sent2vec_inner.pyx

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env cython
```

Comment:
please add a docstrings here too, instruction: https://github.com/RaRe-Technologies/gensim/pull/1944#issuecomment-372909479

Reply:
still  missed `# cython: embedsignature=True `  + need to re-compile after